docs/getting_started.md

```diff
@@ -0,0 +1,110 @@
+    
+    Note over IntegratorRepresentative,ePaymentRepresentative: Manual setup completed
+    
+    Integrator ->> ePaymentPlatform: Generate access token.
+    ePaymentPlatform ->> ePaymentPlatform: Verify request 
+    Note right of ePaymentPlatform: Matching ClientId and performing <br/> a bCrypt hash on the secret
+    ePaymentPlatform -->> Integrator: Return Access token
+```
+
+### Checklist for information exchange
+
+| Information item(s)      | Description                                                                                                                                   |
+|--------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| ClientId                 | Generated by the ePayment Platform and returned to Integrator                                                                                 |
+| CallbackUrl              | Defined by Integrator, will be the prefix address all Callback requests are sent. Peruse our [Callback](#callbacks) section for more context. |
+| ClientSecret             | A bCrypt secret that is **kept secret at the Integrator** and used to generate access tokens.                                                 |
+| ClientSecret bCrypt Hash | A bCrypt secret hash that is sent to the ePaymentPlatform and configured to the Integrators clientID                                          |
+| Authentication Provider  | Inform the EPP team which Authentication Provider you will be utilizing                                                                       |
+| ISS                      | Once the your profile is set up your will receive the Issuer ID corresponding to the your Integrator or Authentication Provider profile.      |
+| EPP public key           | Is sent by EPP during setup, needed to encrypt parts of requests.                                                                             |
+| Token Requestor Name     | Is sent by EPP during setup, needs to be part of enrollment requests..                                                                        |
+
+
+## Integration guidelines.
+
+This section contains general guidelines for integrating with the EPP.
+
+### Context ID 
+All requests support an `X-Correlation-Id` header which can be used to correlate requests and responses. This is especially useful if you always ensure to set this header to a unique value for each request.
+The `X-Correlation-Id` is returned in the corresponding callback, allowing you an additional mechanism to correlate the callback with the original request. It is *very* highly recommended to use this header for enabling traceability and support. 
+
+### MessageId
+The system acts idempotent on any `messageId`. It is *very* highly recommended that you use a robust UUID generator to ensure that each request has a unique `messageId`. 
```

Comment:
Should we change message id and context id requirements level to "absolutely must" or so? And uniqueness should be "a must" too?